### PR TITLE
Issue/translations option calendar

### PR DESCRIPTION
**Pull request by armandduijn**
Fixed translations according to the proposed change from the email of the GEWIS board. Also fixed some wording in the proposed changes. The use of "optie kalendar" and "optiekalender" wasn't consistent. It should be without the space.

### CODE REVIEW

File: module/Activity/view/activity/activity-calendar/index.phtml [hunk 1]
```diff
@@ -12,12 +12,13 @@ $form->prepare();
 <div class="container option-calendar">
     <div class="row">
         <h1><?= $this->translate('Option calendar') ?></h1>
-        <?= $this->translate('Welcome on the page of the digital Option Calendar. In the calendar below you can place options for GEWIS activities, just like this was done before on the paper version of the option calendar. The red dots indicate activities that are already in the GEWIS agenda, the blue dots are options for activities. If you insert an option, please give it a clear name, such that it is clear where the option is for.') ?>
+        <?= $this->translate('Welcome on the page of the digital Option Calendar. In the calendar below you can place options for GEWIS activities, just like you were used to do in the paper version of the option calendar before. The red dots indicate activities that are already fixed in the GEWIS agenda, the blue dots are options for activities. If you insert an option, please give it a clear name, so it is clear where the option is for.') ?>
```
Comment (gafderks) [inline module/Activity/view/activity/activity-calendar/index.phtml]: "so it is clear where the option is for", seems that where should be what. Nice Dunglish though.

Comment (armandduijn) [inline module/Activity/view/activity/activity-calendar/index.phtml]: Don't shoot the messenger. I just copied it from the email.

**Comment by armandduijn** [comment]
@jszanto Could you check the translation files? When I generated them with `./translate-helper` and `msgmege` (for nl and en) it created a lot of changes (see additions, deletions of PR). Not sure if this is normal.

**Comment by jszanto** [comment]
@armandduijn I think it is okay, since it just regernates the whole file. If you fix that grammar bug which @gafderks pointed out then this can be merged.

File: module/Activity/view/activity/activity-calendar/index.phtml (hunk 1 above)
Comment (armandduijn) [inline module/Activity/view/activity/activity-calendar/index.phtml]: Should I change it from
> so it is clear where the option is for
to
> so it is clear what the option is for

**Pull request merged**